learndash-course-grid/course_list_template.php

```diff
@@ -0,0 +1,254 @@
+    $class .= '';
+    $course_class .= ' learndash-available learndash-complete';
+    $ribbon_text = __( 'Completed', 'learndash-course-grid' );
+} elseif ( $price_type == 'open' && empty( $ribbon_text ) ) {
+    if ( is_user_logged_in() && ! $is_completed ) {
+        $class .= ' ribbon-enrolled';
+        $course_class .= ' learndash-available learndash-incomplete';
+        $ribbon_text = __( 'Enrolled', 'learndash-course-grid' );
+    } elseif ( is_user_logged_in() && $is_completed ) {
+        $class .= '';
+        $course_class .= ' learndash-available learndash-complete';
+        $ribbon_text = __( 'Completed', 'learndash-course-grid' );
+    } else {
+        $course_class .= ' learndash-available';
+        $class .= ' ribbon-enrolled';
+        $ribbon_text = '';
+    }
+} elseif ( $price_type == 'closed' && empty( $price ) ) {
+    $class .= ' ribbon-enrolled';
+    $course_class .= ' learndash-available';
+
+    if ( $is_completed ) {
+        $course_class .= ' learndash-complete';
+    } else {
+        $course_class .= ' learndash-incomplete';
+    }
+
+    if ( is_numeric( $price ) ) {
+        $ribbon_text = $price_text;
+    } else {
+        $ribbon_text = '';
+    }
+} else {
+    if ( empty( $ribbon_text ) ) {
+        $class .= ! empty( $course_options['sfwd-courses_course_price'] ) ? ' price_' . $currency : ' free';
+        $course_class .= ' learndash-not-available learndash-incomplete';
+        $ribbon_text = $price_text;
+    } else {
+        $class .= ' custom';
+        $course_class .= ' learndash-not-available learndash-incomplete';
+    }
+}
+
+/**
+ * Filter: individual course ribbon text
+ *
+ * @param string $ribbon_text Returned ribbon text
+ * @param int    $course_id   Course ID
+ * @param string $price_type  Course price type
+ */
+$ribbon_text = apply_filters( 'learndash_course_grid_ribbon_text', $ribbon_text, $course_id, $price_type );
+
+if ( '' == $ribbon_text ) {
+    $class = '';
+}
+
+/**
+ * Filter: individual course ribbon class names
+ *
+ * @param string $class     	 Returned class names
+ * @param int    $course_id 	 Course ID
+ * @param array  $course_options Course's options
+ * @var string
+ */
+$class = apply_filters( 'learndash_course_grid_ribbon_class', $class, $course_id, $course_options );
+
+/**
+ * Filter: individual course container class names
+ *
+ * @param string $course_class   Returned class names
+ * @param int    $course_id 	 Course ID
+ * @param array  $course_options Course's options
+ * @var string
+ */
+$course_class = apply_filters( 'learndash_course_grid_course_class', $course_class, $course_id, $course_options );
+
+$thumb_size = isset( $shortcode_atts['thumb_size'] ) && ! empty( $shortcode_atts['thumb_size'] ) ? $shortcode_atts['thumb_size'] : 'course-thumb';
+
+ob_start();
+?>
+<div class="ld_course_grid col-sm-<?php echo $smcol; ?> col-md-<?php echo $col; ?> <?php echo esc_attr( $grid_class ); ?>">
```

Comment:
can we include class="nhsuk-card nhsuk-card--clickable" here in some way so it inherits keyboard tab improvements from DI-2788